### PR TITLE
Impromvenets/#150

### PR DESCRIPTION
**Pull request by GalochkinKirov**
https://github.com/OAGr/foretold/issues/150

### CODE REVIEW

File: server/src/authorizers/permissions.js [hunk 1]
```diff
@@ -3,24 +3,36 @@ const { shield, allow, and, or, not } = require('graphql-shield');
...
+const { measurableIsOwnedByCurrentAgent } = require('./measurables');
+const { measurableIsArchived } = require('./measurables');
+const { botBelongsToCurrentUser } = require('./bots');
 
 const rulesChannel = {
   Query: {},
   Mutation: {
-    channelUpdate: and(isAuthenticated, isAdmin),
+    channelUpdate: and(
+      isAuthenticated,
```
Comment (OAGr) [inline server/src/authorizers/permissions.js]: can we change this to ``currentUserIsAuthenticated``

File: server/src/authorizers/permissions.js [hunk 2]
```diff
@@ -3,24 +3,36 @@ const { shield, allow, and, or, not } = require('graphql-shield');
...
+      isAdmin,
```
Comment (OAGr) [inline server/src/authorizers/permissions.js]: can we change ``isAdmin`` to ``currentUserIsAdmin``?

File: server/src/authorizers/permissions.js [hunk 3]
```diff
@@ -47,16 +62,30 @@ const rulesChannelMemberships = {
 const rulesMeasurables = {
   Query: {},
   Mutation: {
-    measurementCreate: and(isAuthenticated, or(isChannelPublic, or(isAdmin, isViewer))),
-    measurableArchive: and(isAuthenticated, measurables.isOwner, not(measurables.isArchived)),
-    measurableUnarchive: and(isAuthenticated, measurables.isOwner, measurables.isArchived),
-    measurableUpdate: and(isAuthenticated, measurables.isOwner),
+    measurementCreate: and(
+      isAuthenticated,
+      or(isChannelPublic, or(isAdmin, isViewer)),
```
Comment (OAGr) [inline server/src/authorizers/permissions.js]: plz change ``isViewer`` to ``currentUserIsViewer``

File: server/src/authorizers/permissions.js [hunk 4]
```diff
@@ -83,7 +112,10 @@ const rules = {
     channelCreate: isAuthenticated,
     userUpdate: isAuthenticated,
     seriesCreate: and(isAuthenticated, or(isChannelPublic, isAdmin)),
-    measurableCreate: and(isAuthenticated, or(isChannelPublic, or(isAdmin, isViewer))),
+    measurableCreate: and(
+      isAuthenticated,
+      or(isChannelPublic, or(isAdmin, isViewer)),
```
Comment (OAGr) [inline server/src/authorizers/permissions.js]: Can we change "isChannelPublic" to "channelIsPublic", to go along with the others?

File: server/src/authorizers/permissions.js (hunk 1 above)
Comment (GalochkinKirov) [reply server/src/authorizers/permissions.js]: Updated, cool.

File: server/src/authorizers/permissions.js (hunk 2 above)
Comment (GalochkinKirov) [reply server/src/authorizers/permissions.js]: Updated.

File: server/src/authorizers/permissions.js (hunk 3 above)
Comment (GalochkinKirov) [reply server/src/authorizers/permissions.js]: Updated.

File: server/src/authorizers/permissions.js (hunk 4 above)
Comment (GalochkinKirov) [reply server/src/authorizers/permissions.js]: Updated.

**Comment by GalochkinKirov** [comment]
Thanks, like semantics too, updated.

**Pull request merged**